packages/cli/src/commands/split/index.ts

```diff
@@ -41,23 +42,16 @@ export async function handleSplit (argv: {
   printExecutionTime('split', startedAt, entrypoint!);
 }
 
-function splitDefinition(openapi: Oas3Definition, openapiDir: string) {
+function splitDefinition(openapi: Oas3Definition | Oas3_1Definition, openapiDir: string) {
   fs.mkdirSync(openapiDir, { recursive: true });
-  const pathsDir = path.join(openapiDir, PATHS);
-  fs.mkdirSync(pathsDir, { recursive: true });
 
   const componentsFiles: ComponentsFiles = {};
-  iteratePaths(openapi, pathsDir, openapiDir);
   iterateComponents(openapi, openapiDir, componentsFiles);
+  iteratePathItems(openapi.paths, openapiDir, path.join(openapiDir, 'paths'), componentsFiles);
```

Comment:
I believe it was this way even in your PR - https://github.com/Redocly/openapi-cli/commit/000f45acb34ecaa05e80da50616761c4ba3de3e5

Reply:
hmmm, weird, it used to work so I'm just worried something went wrong....

Reply:
I believe this happens because `traverseDirectoryDeep` is now called inside `iteratePathItems` (and we call `replace$Refs` there).
Previuosly, `traverseDirectoryDeep` was called after `iterateComponents` and this way refs were resolved correctly.
We can change this again, but I believe it's easier to have iterateComponents above. I believe it doesn't break any logic.
@RomanHotsiy

Reply:
Just test it with some definitions and I'm good.